src/posts/keymaps/peterjc-new_stickney.md

```diff
@@ -0,0 +1,24 @@
+---
+author: peterjc
+baseLayouts: ["QWERTY"]
+firmwares: ["Karabiner-Elements"]
+hasHomeRowMods: false
+hasLetterOnThumb: false
+hasRotaryEncoder: false
+isAutoShiftEnabled: false
+isComboEnabled: false
+isSplit: false
+isTapDanceEnabled: false
+keybindings: []
+keyboard: ANSI
```

Comment:
I don't see existing generic entries for JIS or ISO, both of which would also be fine.

Reply:
This is a single value field, and ANSI has less keys than ISO or JIS, so leaving it as ANSI as per the field description notes.